deploy/crds/k8s.cni.cncf.io_networkattachmentdefinitions_crd.yaml

```diff
@@ -1,14 +1,14 @@
-apiVersion: apiextensions.k8s.io/v1
+apiVersion: apiextensions.k8s.io/v1beta1
```

Comment:
This file seems not used in subscription deployment.
Is it because CNO is still using v1beta1, so we don't change this to version 1? I noticed it was changed to v1 in the second commit, and changed back to v1beta1 in this commit.

Reply:
yes I fix it in the last commit on this PR thanks!